README.md

```diff
@@ -2,26 +2,37 @@
-
-
-### Linux/Mac - Unit Tests
-|Branch|Ubuntu16|macOS|
+### Desktop Unit Tests
+|Branch|Debug x86|Debug x64|Release x86|Release x64|Spanish|
+|:--:|:--:|:--:|:--:|:--:|:--:|
+**master**|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-CI?label=build&branchname=master&jobname=Windows_Desktop_Unit_Tests&configuration=debug_32)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=15&branchname=master)|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-CI?label=build&branchname=master&jobname=Windows_Desktop_Unit_Tests&configuration=debug_64)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=15&branchname=master)|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-CI?label=build&branchname=master&jobname=Windows_Desktop_Unit_Tests&configuration=release_32)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=15&branchname=master)|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-CI?label=build&branchname=master&jobname=Windows_Desktop_Unit_Tests&configuration=release_64)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=15&branchname=master)|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-CI?label=build&branchname=master&jobname=Windows_Desktop_Spanish_Unit_Tests&configuration=)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=15&branchname=master)|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-CI?label=build&branchname=master&jobname=Windows_Determinism_Test&configuration=)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=15&branchname=master)|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-CI?label=build&branchname=master&jobname=Windows_Correctness_Test&configuration=)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=15&branchname=master)|
```

Comment:
These links all go to the same place, which lists the problems across _all_ builds in that `roslyn-CI` run with no immediate indication of what's what. So when you click on an individual queue here you might think the problems you then see are for that queue, but they may or may not be. If we can't make per-queue links (I don't see how but haven't tried hard), consider adding `&view=logs` to the end of the link URLs so you can see the overview and click the queue you were interested in.